### PR TITLE
Ignore tests on ci

### PR DESCRIPTION
**Pull request by fommil**
replaces #1692

### CODE REVIEW

**Comment by fommil** [comment]
closes #1662

**Comment by fommil** [comment]
and try to elleviate #1683

File: project/EnsimeBuild.scala [hunk 1]
```diff
@@ -234,7 +234,7 @@ object EnsimeBuild {
 
   private def akkaVersion: Def.Initialize[String] = Def.setting {
     CrossVersion.partialVersion(scalaVersion.value) match {
-      case Some((2, minor)) if minor >= 11 => "2.4.14"
+      case Some((2, minor)) if minor >= 11 => "2.4.16"
```
Comment (sugakandrey) [inline project/EnsimeBuild.scala]: Yay! No more noisy logging :tada:

**Comment by fommil** [comment]
hmm, I see the same windows hanging nonsense that you seen.

**Comment by sugakandrey** [comment]
yeah, looks like windows doesn't like test run's arguments at all. I think even 
```
it:test-only * -- -oD
```
behaved the same way for me.

**Comment by fommil** [comment]
could be the escaping rules, maybe a dash or the asterix is being expanded...

**Comment by fommil** [comment]
this used to work `it:testOnly -- -l Debugger`. I think the `*` is both superflous and confusing on Windows.

**Comment by sugakandrey** [comment]
@fommil https://github.com/sbt/sbt-launcher-package/pull/122 check this out

**Comment by fommil** [comment]
even if the CIs still timeout, I'll merge this... it's an improvement already.

**Comment by fommil** [comment]
file watchers hanging on windows... I'll ignore them for now

**Comment by fommil** [comment]
That seems to have helped. Might be able to remove the ignore tags

**Comment by fommil** [comment]
Just server startup causing trouble now

**Pull request merged**